lib/src/test_options/test_options_ext.dart

```diff
@@ -32,25 +35,31 @@ extension IterableTestParametersEx<T extends Iterable<dynamic>> on T {
       );
 }
 
-extension TestParametersEx<T extends Object> on T {
+/// Extension on [Object] to apply extra test options for a specified
+/// test value.
+extension TestParametersEx on Object {
   /// Applied extra test options for a specified test value.
   /// For example:
   /// ```dart
   /// [
   ///   4,
-  ///   5.withTestOptions(skip: true),
+  ///   5.options(skip: true),
   /// ]
   /// ```
-  ValueWithTestOptions withTestOptions({
+  ValueWithTestOptions options({
+    CustomDescriptionBuilder? customDescriptionBuilder,
+    String? testOn,
     dart_test.Timeout? timeout,
     dynamic skip,
     dynamic tags,
     Map<String, dynamic>? onPlatform,
     int? retry,
   }) =>
-      ValueWithTestOptions(
-        [this],
-        TestOptions(
+      (
+        values: [this],
```

Comment:
Why is this an array? 🤔

Reply:
Because this extension is used on a list of single values. For example:
```dart
parameterizedTest('singel values', [1,2,3], (int value) {...});
```

The `ValueWithTestOptions` expects a list of values. Also the `Function.apply` expects a list of arguments even if there is only 1 argument.

Reply:
Aah got it!